index.js

```diff
@@ -84,6 +82,30 @@ class K8sVMExecutor extends Executor {
                     throw new Error(`Failed to create pod: ${JSON.stringify(resp.body)}`);
                 }
 
+                return resp.body.metadata.name;
+            })
+            .then(podname => new Promise(resolve => setTimeout(resolve, 3000))
```

Comment:
Neat little delay trick, but should move the magic number to a constant somewhere else.

Reply:
use `RETRYDELAY` instead of `3000`